seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -872,7 +872,7 @@
         "capitalizeString(\"hello!\");"
       ],
       "tests": [
-        "assert(code.match(/import\\s+\\{\\s?capitalizeString\\s?\\}\\s+from\\s+\"string_functions\"/ig)"
+        "assert(code.match(/import\\s*\\{\\s*capitalizeString\\s*\\}\\s*from\\s*\\\"string_functions\\\"/ig), 'message: Import <code>capitalizeString</code> function from the <code>\"string_functions\"</code> file');"
```

Comment:
Yes, was working on that and must have been on the wrong branch. I'll update this tonight. 

I don't think the filename needs to be changed.
Thanks @systimotic